modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/components/card/CardList.es.js

```diff
@@ -94,7 +94,9 @@ export default ({data, fields}) => {
 			...fieldTypes[field.type],
 		};
 
-		const chart = chartFactory(field, values, summary, totalEntries);
+		const chartContent = {field, summary, totalEntries, values};
+
+		const chart = chartFactory(chartContent);
```

Comment:
@alinedoleron, is there any special reason for it?

Reply:
Bui asked me to create an object, because the list of arguments were increasing a lot.